model/workflow.go

```diff
@@ -515,7 +517,14 @@ type dataInputSchemaUnmarshal DataInputSchema
 // UnmarshalJSON implements json.Unmarshaler
 func (d *DataInputSchema) UnmarshalJSON(data []byte) error {
 	d.ApplyDefault()
-	return util.UnmarshalPrimitiveOrObject("dataInputSchema", data, &d.Schema, (*dataInputSchemaUnmarshal)(d))
+	if data[0] == '"' && len(data) > 0 {
```

Comment:
@spolti @ricardozanini 

You can change `UnmarshalJSON`. Because according to the specification, when it is `string` it loads the json schema from external resource.

```go
func (d *DataInputSchema) UnmarshalJSON(data []byte) error {
       err := util.UnmarshalObject("dataInputSchema", data, (*dataInputSchemaUnmarshal)(d))
       if err != nil {
               return err
       }

       if d.Schema.Type == String {
               data, err := util.LoadExternalResource(d.Schema.StringValue)
               if err != nil {
                       return err
               }
               return util.UnmarshalObject("schema", data, d.Schema)
        }
}
```

Also change the tests when `string` to something like `file://testdata/datainputschema.json`

Obs.: The `Object struct` implements the `UnmarshalJSON`